### PR TITLE
dev-python/prov: version bump, solves Bug 628054

### PR DESCRIPTION
**Pull request by TheChymera**
Package-Manager: Portage-2.3.6, Repoman-2.3.3

Solves https://bugs.gentoo.org/show_bug.cgi?id=628054

Tests now work, except on python 3.*, because our `media-gfx/pydot` package is not marked as compatible. Looking to [update that](https://pypi.python.org/pypi/pydot) as well, soon.

### CODE REVIEW

File: dev-python/prov/prov-1.5.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+HOMEPAGE="https://pypi.python.org/pypi/prov"
+SRC_URI="mirror://pypi/p/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+RDEPEND="
+	dev-python/pydotplus[${PYTHON_USEDEP}]
```
Comment (gktrk) [inline dev-python/prov/prov-1.5.1.ebuild]: The source has `import pydot` here: https://github.com/trungdong/prov/blob/master/prov/dot.py#L22. How does pydotplus work with it? Shouldn't we depend on pydot and drop python3 support?

Comment (TheChymera) [inline dev-python/prov/prov-1.5.1.ebuild]: no way are we dropping python 3 support :) https://github.com/gentoo/gentoo/pull/5478

Comment (gktrk) [inline dev-python/prov/prov-1.5.1.ebuild]: In the light of #5478, I'm guessing we need to change our depend to `dev-python/pydot` and it should all work right?

**Comment by TheChymera** [comment]
@gktrk yes, it works with `=pydot-1.0.28 `

**Comment by gktrk** [comment]
Merged in https://github.com/gentoo/gentoo/compare/8ae28c0...2759328. Thanks!

Minor note, your email address in the commit author field does not match the email address in the metadata.xml. You should fix it if possible.

**Pull request closed without merging**